pkgs/applications/audio/abcde/default.nix

```
@@ -1,16 +1,23 @@
 { lib, stdenv, fetchurl, libcdio-paranoia, cddiscid, wget, which, vorbis-tools, id3v2, eyeD3
 , lame, flac, glyr
 , perlPackages
+, fetchFromGitHub
 , makeWrapper }:
 
-let version = "2.9.3";
+let version = "2.9.4pr4";
```

```suggestion
let version = "2.9.3-unstable-2023-04-23";
```
Is the nixpkgs convention. Sorry for the late review.